dask_jobqueue/core.py

```diff
@@ -477,7 +477,9 @@ def __init__(
         if interface is None:
             interface = dask.config.get("jobqueue.%s.interface" % config_name)
         if scheduler_options is None:
-            scheduler_options = {}
+            scheduler_options = dask.config.get(
+                "jobqueue.%s.scheduler-options" % config_name, {}
```

Comment:
Can you explain me why the `{}` is needed ? I would have thought that since `scheduler-options` is in `jobqueue.yaml` it should not be needed but the CI was failing without the `{}` so I must be missing something ...

Reply:
The "problem" is that the configuration dictionaries used for testing are all set up explicitely, and thus won't inherit from `jobqueue.yaml`, see [here](https://github.com/kathoef/dask-jobqueue/blob/9bc806855364b0fe143271b366ea82601edda31a/dask_jobqueue/tests/test_slurm.py#L193) for an example. (Copying that approach is actually how I came up with this `generic_cluster_conf` dictionary.)

Instead of having
```
dask.config.get("jobqueue.%s.scheduler-options" % config_name, {})
```
it would also be possible to add `scheduler-options: {}` for each of these configuration dictionaries used for testing of the specific clusters.

However, then we would actually require the existence of a `scheduler-options` entry for standalone configurations, which would break already existing user configurations.

Reply:
> However, then we would actually require the existence of a scheduler-options entry for standalone configurations, which would break already existing user configurations.

OK fair enough, for not breaking backward-compatibility, let's keep the `.get`